Test fails upstream too, see https://github.com/pytorch/pytorch/issues/147853
> RuntimeError: Expected to find ".to(" but did not find it

Author: Alexander Grund (TU Dresden)
--- a/test/inductor/test_pattern_matcher.py
+++ b/test/inductor/test_pattern_matcher.py
@@ -389,6 +389,7 @@ class TestPatternMatcher(TestCase):
         }
     )
     @unittest.skipIf(not IS_BIG_GPU, "templates require big gpu")
+    @unittest.skip("Fails")
     def test_mixed_mm_exhaustive_dtypes(self):
         def fn(a, b):
             return torch.mm(a, b.to(a.dtype))
